wire-ios/Wire-iOS/Sources/Authentication/Interface/ViewControllers/AuthenticationHostingController.swift

```diff
@@ -50,6 +50,10 @@ final class AuthenticationHostingController<Content: View>: UIHostingController<
                 )
             case .exitFlowRequested:
                 self?.selectAccount()
+            case let .logoutRequested(deleteData):
+                authenticationCoordinator?.eventResponderChain.handleEvent(
+                    ofType: .logoutRequested(deleteData: deleteData)
+                )
```

Comment:
This is the bridge receiving the outbound event and connecting to the legacy authentication system.